src/stores/PaymentStore.ts

```diff
@@ -109,10 +109,14 @@ export const usePaymentStore = defineStore('payment',{
           console.log('id undefined')
```

Comment:
Something I thought of a little while ago, if we could use Pop-Ups, Toasts or something else to let the user know she has to check the AGB's button instead of pushing something to console.log ?

Also if `this.transactionID === ""` = true, shouldn't we do something like "Connection to server not stable right now. Please try again later" error handling and kind of going the same way in showing it the user via a Toast, Pop-Up... ?

![image](https://github.com/augustin-wien/augustin-frontend/assets/38591956/20a60e58-4b9f-4aaa-819e-3c77d6ca7032)

Reply:
I created an issue for this demand. So we can still merge it now.